sdk/botservice/arm-botservice/test/botservice_examples.ts

```diff
@@ -57,77 +54,11 @@ describe("Botservice test", () => {
     await recorder.stop();
   });
 
-  it("bots create test", async function () {
-    const res = await client.bots.create(resourceGroup, botresourceName, {
-      sku: {
-        name: "S1"
-      },
-      etag: "etag1",
-      tags: {
-        tag1: "value1",
-        tag2: "value2"
-      },
-      kind: "sdk",
-      properties: {
-        description: "The description of the bot",
-        developerAppInsightsApiKey: "w24iw5ocbhcig71su7ibaj63hey5ieaozeuwdv2r",
-        developerAppInsightKey: "59513bad-10a7-4d41-b4d0-b1c34c6af52a",
-        developerAppInsightsApplicationId: "cf03484e-3fdb-4b5e-9ad7-94bde32e5a2b",
-        displayName: "this is a test bot",
-        endpoint: "https://bing.com/messages/",
-        msaAppId: "41a220b9-6571-4f0b-bbd2-43f1c1d82f51"
```

Comment:
remove this test case considering it requires an app register but we don't have a permission for that.